File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.js
function solution(rank, attendance) {
    let answer = [];
    rank.unshift(0)
    attendance.unshift(0)
    for (let i = 0; i < rank.length; i++) {
        if(rank.indexOf(i) && attendance[rank.indexOf(i)]) {
            answer.push(rank.indexOf(i))
        }
    }
    return (10000 * (answer[0] - 1) ) + (100 * (answer[1] - 1) ) + (answer[2] - 1) 
}

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
    ranked_students_info = []
    for i in range(len(rank)):
        ranked_students_info.append((rank[i], i, attendance[i]))

    ranked_students_info.sort()

    selected_students_original_indices = []
    for r, original_idx, attended in ranked_students_info:
        if attended:
            selected_students_original_indices.append(original_idx)
        if len(selected_students_original_indices) == 3:
            break

    return (10000 * selected_students_original_indices[0]) + \
           (100 * selected_students_original_indices[1]) + \
           (selected_students_original_indices[2])